3 reports.sql

```diff
@@ -0,0 +1,20 @@
+--1) I need to know if I have any repeat customers and what they rented.
+select CustomerName = concat(m.CustomerFirstName, ' ', m.CustomerLastName)
+ from MusicRental m
+ group by concat(m.CustomerFirstName, ' ', m.CustomerLastName)
+ having count(concat(m.CustomerFirstName, ' ', m.CustomerLastName)) > 1
+ order by count(concat(m.CustomerFirstName, ' ', m.CustomerLastName))
+--2) I need to know the most popular category of instrument that has ever been rented.
+select top 1 m.Category
+from MusicRental m
+group by m.Category
+order by COUNT(m.Category) desc
+--3) I need to know which instrument (that is not being currently rented) made the most profit.
```

Comment:
The report just asks to return the profit and instrument, the report doesn't ask for the other data that you are returning

Reply:
Fixed mistake